src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1686,7 +1714,7 @@ public Response submitDatasetVersionToArchive(@PathParam("id") String dsid, @Pat
 			// DataverseRequest and is sent to the back-end command where it is used to get
 			// the API Token which is then used to retrieve files (e.g. via S3 direct
 			// downloads) to create the Bag
-            session.setUser(au);
+            session.setUser(au); // TODO: Stop using session. Use createDataverseRequest instead.
```

Comment:
Is this still a TODO?

Reply:
This is a future TODO that I discussed with @qqmyers . I didn't want to mess with this SubmitToArchive (and ask for QA to re-test it) but it's somewhat of session.setUser, I feel. As I indicated, createDataverseRequest is what we usually do in this case.